db/migrate/20221104024827_add_status_to_recipient.rb

```diff
@@ -0,0 +1,21 @@
+class AddStatusToRecipient < ActiveRecord::Migration[7.0]
+
+  def up
+    create_enum :recipient_status, %w(imported consent_error api_error api_success delivery_error delivery_success)
```

Comment:
How is `consent_error` used/interpreted?

Reply:
It isn't yet - that'll be in a separate change. I'll remove it from here for now, since we might make some different decisions about how to represent this case when we actually implement consent checking.